source/stylesheets/site.css.scss

```diff
@@ -358,3 +331,32 @@ main {
     }
   }
 }
+
+@import 'footer';
+
+@media screen and (max-width: 36em) {
```

Comment:
Where's this magic number `36em` coming from? Might be good to create a variable for it, I see it's used in other places as well.

Reply:
I have no clue - however there is more magic :wizard: , since we having these breakpoints:

```
@media screen and (max-width: 36em)
@media screen and (max-width: 52em)
@media screen and (max-width: 62em) 
```

I would like to keep that outside of the scope of this pull request, since I am thinking that the css needs a overhaul (hint: see the `@import` statements in the site.css.scss).